flang/include/flang/Common/CommandLineOpts.h

```diff
@@ -0,0 +1,60 @@
+//===-- CommandLineOptions.h -- shared command line options -----*- C++ -*-===//
```

Comment:
I agree with Slava that this does not belong to Common, CommandLineOpts, putting it in the same library as  Pipelines.cpp makes sense to me.

nit: "CommandLineOptions.h" -> "CommandLineOpts.h"

Reply:
Done.